configs/lvis/README.md

```diff
@@ -11,6 +11,11 @@
 ```
 
 ## Common Setting
+* Please follow [install guide](../../docs/install.md#install-mmdetection) to install open-mmlab forked cocoapi first.
+* Run following scripts to install our forked lvis-api.
+    ```
+    pip install "git+https://github.com/open-mmlab/cocoapi.git#subdirectory=lvis"
+    ```
```

Comment:
We can check the version of lvis and raise an error if users installed the official release.